Library/Formula/awscli.rb

```diff
@@ -100,7 +103,7 @@ def caveats; <<-EOS.undent
       #{HOMEBREW_PREFIX}/share/awscli/examples
 
     Add the following to ~/.bashrc to enable bash completion:
-      complete -C aws_completer aws
+      complete -C #{HOMEBREW_PREFIX}/etc/bash_completion.d/aws_completer aws
 
     Add the following to ~/.zshrc to enable zsh completion:
       source #{HOMEBREW_PREFIX}/share/zsh/site-functions/_aws
```

Comment:
I think Homebrew should auto-generate zsh and bash completion caveats; if these messages get repeated below after installation we can remove these.